translator/translate/otel/exporter/awsemf/kubernetes.go

```diff
@@ -99,7 +99,7 @@ func getPodMetricDeclarations(conf *confmap.Conf) []*awsemfexporter.MetricDeclar
 		podMetricDeclarations[0].Dimensions = append(podMetricDeclarations[0].Dimensions, []string{"FullPodName", "PodName", "Namespace", "ClusterName"})
 		selectors = append(selectors, []string{"pod_number_of_container_restarts", "pod_number_of_containers", "pod_number_of_running_containers",
 			"pod_status_ready", "pod_status_scheduled", "pod_status_running", "pod_status_pending", "pod_status_failed", "pod_status_unknown",
-			"pod_status_succeeded", "pod_memory_request", "pod_memory_limit", "pod_cpu_limit", "pod_cpu_request",
+			"pod_status_succeeded", "pod_memory_request", "pod_memory_limit", "pod_cpu_limit", "pod_cpu_request", "pod_cpu_usage_total", "pod_memory_working_set",
```

Comment:
nit: keep the memory metric next to the other memory metrics so its easier to find when someone is reading this list.

```
"pod_memory_request", "pod_memory_limit", "pod_memory_working_set", "pod_cpu_limit", "pod_cpu_request", "pod_cpu_usage_total",
```

Reply:
sure, will fix it in next PR